pkgs/top-level/all-packages.nix

```diff
@@ -18932,14 +18932,14 @@ in
   ### DEVELOPMENT / PERL MODULES
 
   perlInterpreters = callPackages ../development/interpreters/perl {};
-  inherit (perlInterpreters) perl530 perl532 perldevel;
+  inherit (perlInterpreters) perl532 perl534 perldevel;
 
-  perl530Packages = recurseIntoAttrs perl530.pkgs;
```

Comment:
Do we want to add an alias for the removed version?

Reply:
Hm, I don't think so. If someone uses `perl530` rather than `perl` they would expect to get 5.30.x?

Reply:
Yep, and we already remove the `withPerl530` predicate above so an alias would probably end up undefined.

This also makes it more consistent with https://metacpan.org/pod/perlpolicy#MAINTENANCE-AND-SUPPORT (supporting only the last 2 Perl stable releases.)